modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/QueryMemoryTracker.java

```diff
@@ -179,21 +182,36 @@ private boolean onQuotaExceeded() {
     private void releaseFromParent() {
         long toReleaseFromParent = reservedFromParent - reserved;
 
-        parent.released(toReleaseFromParent);
+        parent.release(toReleaseFromParent);
 
         reservedFromParent -= toReleaseFromParent;
 
         assert reservedFromParent >= 0 : reservedFromParent;
     }
 
     /** {@inheritDoc} */
-    @Override public synchronized long memoryReserved() {
+    @Override public synchronized long reserved() {
```

Comment:
It looks like there is no need to mark method with 'synchronized' just to read volatile field